office/meetingbar/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        leits MeetingBar 1.3 v
+name                meetingbar
+
+platforms           darwin
+categories          office aqua
+supported_archs     x86_64
+license             Apache-2
+
+homepage            https://meetingbar.onrender.com/
+
+description         Your next meeting always before your eyes in macOS menu \
+                    bar.
+
+long_description    MeetingBar is a menu bar app for your calendar meetings. \
+                    Integrated with Google Meet and Zoom so you can quickly \
+                    join meetings from event or create ad hoc meeting.
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           rmd160  18d24d964eaa8148c8d8a241f50584e82a633af2 \
+                    sha256  ab1309ffa0f00ca590951d8ae827c2c54180ea48b6b327e209d9261427361097 \
+                    size    301497
+
+github.tarball_from releases
+
+distname            MeetingBar
+use_configure       no
+use_dmg             yes
+
+if {${os.major} < 19} {
+    return -code error "${name} runs on macOS 10.15+"
+}
```

Comment:
The return statement here should be put inside a pre-fetch block, which is the generally acceptable point in the port processing we prefer these sorts of errors to be issued.

Please also add

known_fail yes

Which prevents build attempts on the buildbots for failing platforms.

See e.g.

https://github.com/macports/macports-ports/blob/370dafef40932d2a53303c40d4c09ae3f6f28c8b/science/root6/Portfile#L209

As a randomly picked example of how it should generally look